package.json

```diff
@@ -30,5 +33,8 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
+  },
+  "devDependencies": {
+    "react-admin": "^3.0.4"
```

Comment:
react admin is a dependency not a dev dependency, so uninstall it and install as a dependency